src/Calendar/components/DayGrid/utils/useTotalEventsToShow.js

```diff
@@ -16,46 +12,49 @@ const useTotalEventsToShow = () => {
   // even if there are no events show right away
   const [totalEventsToShow, setTotalEventsToShow] = useState(0);
 
-  useEffect(() => {
-    if (eventRef.current) {
-      const currentEventHeight = eventRef.current.offsetHeight;
-      if (currentEventHeight !== eventHeight) {
-        setEventHeight(currentEventHeight);
-      }
+  /**
+   * Using callbacks instead of refs as detailed here:
+   * https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node
```

Comment:
thx for including this resource, very helpful 👍

Reply:
:100: